uk/strings.po

```diff
@@ -150,8 +153,14 @@ msgid "Activate Account"
 msgstr "Створення акаунту"
 
 #. i18n: keyword: ActivateAccountDescription
-msgid "Your Klei account was successfully created! Please complete the form to activate it and have access to additional features. You can also subscribe to our newsletter to receive information about your favorite game."
-msgstr "Ваш обліковий запис Klei успішно створено! Заповніть форму, щоб активувати його та мати доступ до додаткових функцій. Ви також можете підписатися на нашу розсилку, щоб отримувати інформацію про свою улюблену гру."
+msgid ""
```

Comment:
This PR seems to have done something strange with the formatting of the msgids. They're all broken up into several lines.

Reply:
This was done by Poedit. After opening fresh file I have next popup:

![image](https://user-images.githubusercontent.com/10547729/167849497-fdfac171-4b01-4cdf-8083-3589a99cf564.png)

Reply:
I can try to redo this. Or maybe use another software.